tensorflow/lite/micro/testing/test_bluepill_binary.sh

```diff
@@ -27,30 +27,29 @@
 declare -r ROOT_DIR=`pwd`
 declare -r TEST_TMPDIR=/tmp/test_bluepill_binary/
 declare -r MICRO_LOG_PATH=${TEST_TMPDIR}
-declare -r MICRO_LOG_FILENAME=${MICRO_LOG_PATH}/logs.txt
+declare -r MICRO_LOG_FILENAME=${MICRO_LOG_PATH}logs.txt
 mkdir -p ${MICRO_LOG_PATH}
 
-docker build -t renode_bluepill \
-  -f ${ROOT_DIR}/tensorflow/lite/micro/testing/Dockerfile.bluepill \
-  ${ROOT_DIR}/tensorflow/lite/micro/testing/
+if [ -e $MICRO_LOG_FILENAME ]; then
```

Comment:
This is optional. The aim is to have only most recent files in the `MICRO_LOG_PATH`. Otherwise Renode will move the existing file to `logs.txt.1` (or `logs.txt.2` if it is taken etc.).